pkg/web/form.go

```diff
@@ -24,6 +24,13 @@ type Field struct {
 	Visible          bool    `json:"visible,omitempty"`
 	DisplayWhen      string  `json:"displayWhen,omitempty"`
 	Fields           []Field `json:"fields,omitempty"`
+	Options          []Option `json:"options,omitempty"`
```

Comment:
Please confirm if this is still not working in Go and that I need to look into it!

Reply:
its working